### PR TITLE
Drop redundant pe_working_set_t arguments from libpacemaker functions

### PR DESCRIPTION
**Pull request by kgaillot**
This focuses on exposed functions (some static functions still could be changed)

### CODE REVIEW

**Comment by kgaillot** [comment]
@clumens , I think you'll like this one :) Low priority

File: lib/pacemaker/pcmk_sched_group.c [hunk 1]
```diff
@@ -149,36 +149,44 @@ group_create_actions(pe_resource_t * rsc, pe_working_set_t * data_set)
     for (; gIter != NULL; gIter = gIter->next) {
         pe_resource_t *child_rsc = (pe_resource_t *) gIter->data;
 
-        child_rsc->cmds->create_actions(child_rsc, data_set);
+        child_rsc->cmds->create_actions(child_rsc);
         group_update_pseudo_status(rsc, child_rsc);
     }
 
     op = start_action(rsc, NULL, TRUE /* !group_data->child_starting */ );
```
Comment (clumens) [inline lib/pacemaker/pcmk_sched_group.c:156]: For a future commit, it'd be nice to make the value passed just be the thing that's commented out.

**Review by clumens — COMMENTED** [review]
I'm for it!

File: lib/pacemaker/pcmk_sched_group.c (hunk 1 above)
Comment (kgaillot) [reply lib/pacemaker/pcmk_sched_group.c:156]: child_starting (and this comment) is actually removed in a later PR

**Pull request merged**